server/src/main/kotlin/suwayomi/tachidesk/manga/MangaAPI.kt

```diff
@@ -48,7 +48,7 @@ object MangaAPI {
             post("{sourceId}/filters", SourceController.setFilters)
 
             get("{sourceId}/search", SourceController.searchSingle)
-            post("{sourceId}/search", SourceController.searchSingle)
+            post("{sourceId}/filter", SourceController.filterSingle)
```

Comment:
name it `quick-search` and `quickSearchSingle`

Reply:
Shouldn't the url use `quickSearch`? Looks like API uses camel case for multiple words (see `recentChapters`).

Reply:
It's one instance of camel case here and kebab case over at `GlobalAPI`, usually kebab case is used in urls